keyring/secrets.go

```diff
@@ -0,0 +1,239 @@
+// +build linux
+
+package keyring
+
+import (
+  "github.com/gsterjov/go-libsecret"
+  "fmt"
+  "os"
+  "encoding/json"
+)
+
+const SECRETS_COLLECTION = "awsvault"
```

Comment:
Go style is `secretsCollection` or `SecretsCollection` if it's supposed to be exported.